dashboard/src/components/Config/AppRepoList/AppRepoButton.scss

```diff
@@ -1,6 +0,0 @@
-.modal-close {
```

Comment:
Delete the modal close handcrafted style since our modal already has one (but the icon wasn't being loaded)